code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2383,3 +2444,64 @@
 		M.adjustOxyLoss(2, FALSE)
 		M.adjustBruteLoss(2, FALSE)
 	..()
+
+/datum/reagent/acidic_buffer
+	name = "Strong acidic buffer"
+	description = "This reagent will consume itself and move the pH of a beaker towards acidity when added to another."
+	color = "#fbc314"
+	pH = 0
+	data = list("pH" = 0)
+	var/strength = 20
```

Comment:
Could you document this variable? Give us an idea of what possible strength ranges we're working with?

Reply:
Hopefully that's descriptive enough. It's sort of arbitary but it ensure that pH change respects volume
`///The strength of the buffer where (volume/holder.total_volume)*strength. So for 1u added to 50u the pH will increase by 0.3`

Reply:
Also it's a var so that weaker/stronger buffers can be added in the future.